tests/baselines/reference/user/prettier.log

```diff
@@ -5,7 +5,7 @@ index.js(138,21): error TS2532: Object is possibly 'undefined'.
 src/cli/util.js(262,64): error TS2339: Property 'length' does not exist on type 'Ignore'.
 src/cli/util.js(335,52): error TS2339: Property 'length' does not exist on type 'Ignore'.
 src/cli/util.js(396,46): error TS2345: Argument of type 'null' is not assignable to parameter of type 'number | undefined'.
-src/cli/util.js(403,39): error TS2339: Property 'grey' does not exist on type 'typeof import("E:/Github/TypeScript/node_modules/chalk/types/index")'.
+src/cli/util.js(403,39): error TS2339: Property 'grey' does not exist on type 'typeof import("/opt/vsts/work/1/s/node_modules/chalk/types/index")'.
```

Comment:
why is that absolute path?

Reply:
It's outside of the test directory and I made a mistake and assumed the cloned dir would be names `TypeScript` for other paths.